FreeTAKServer/model/FTSModel/_medevac_ .py

```diff
@@ -1,48 +1,30 @@
 from FreeTAKServer.model.FTSModel.fts_protocol_object import FTSProtocolObject
 #######################################################
-# 
+#
 # _medevac_ .py
-# Python implementation of the Class _medevac_ 
+# Python implementation of the Class _medevac_
 # Generated by Enterprise Architect
 # Created on(FTSProtocolObject):      13-Apr-2020 4(FTSProtocolObject):07(FTSProtocolObject):52 PM
 # Original author: Corvo
-# 
+#
 #######################################################
 
+
 class _medevac_(FTSProtocolObject):
     """the medevac class is used to describe a case of someone in need to be evacuated
     """
-# default constructor  def __init__(self):  
+    # default constructor  def __init__(self):
 
-    __litter = "1" 
-    __freq = "0.0" 
-    __terrain_none = "true" 
-    __zone_prot_selection = "0" 
-    __Title = "" 
-    __Priority = "1" 
-    __medline_remarks = "" 
-    __Security = "" 
-    __routine = "" 
-    __equipment_none = "" 
-    __hlz_marking = "" 
-    __casevac = "" 
+    __litter = "1"
+    __freq = "0.0"
+    __terrain_none = "true"
+    __zone_prot_selection = "0"
+    __Title = ""
+    __Priority = "1"
+    __medline_remarks = ""
+    __Security = ""
+    __routine = ""
+    __equipment_none = ""
+    __hlz_marking = ""
+    __casevac = ""
     __urgent = ""
-
-class _medevac_ :
```

Comment:
This seemed like a duplicate. One `_medevac_` inherited from `FTSProtocolObject` but the other didn't.